libs/core/src/lib/dialog/dialog.component.ts

```diff
@@ -157,7 +157,6 @@ export class DialogComponent extends DialogBase implements OnInit, OnChanges, Af
     /** @hidden */
     ngAfterViewInit(): void {
         super.ngAfterViewInit();
-        this.dialogWindow.nativeElement.focus();
```

Comment:
Why it has been removed?

Reply:
this does not do anything, and not required